File: TXIntegrator.View/Javascript/TXIntegrator/Model/Pianificazioni.js
dojo.require("Jarvix.Core.TableModel");

dojo.provide("TXIntegrator.Model.Pianificazioni");
dojo.declare("TXIntegrator.Model.Pianificazioni", Jarvix.Core.TableModel, {

	Controller: "/TXIntegrator/Controller/TXIntegrator.ashx",

	Data: {
		Id: null,
		IdViaggio: null,
		IdSpedizione: null,
		Stato: null,
		SyncStato: null,
		SyncTask: null,
		SyncData: null
	}

});

File: TXIntegrator.Model/Pianificazioni.cs
using System;
using Ultrapulito.Jarvix.Core;

namespace Volcano.TXIntegrator.Model{

	public partial class Pianificazioni : TableModel {

		#region STORED PROCEDURES
		public override string SP_READ { get { return "TX.__Pianificazioni_Rx"; } }
		public override string SP_WRITE { get { return "TX.__Pianificazioni_Wx"; } }
		#endregion


		#region CAMPI DATABASE
		public int? Id { get; set; }
		public int? IdViaggio { get; set; }
		public int? IdSpedizione { get; set; }
		public string Stato { get; set; }
		public string SyncStato { get; set; }
		public string SyncTask { get; set; }
		public DateTime? SyncData { get; set; }
		#endregion


		#region COSTRUTTORI
		public Pianificazioni(){
			this.Id = null;
			this.IdViaggio = null;
			this.IdSpedizione = null;
			this.Stato = null;
			this.SyncStato = null;
			this.SyncTask = null;
			this.SyncData = null;
		}
		#endregion

	}

}

